docs/cvl/expr.md

```diff
@@ -289,8 +298,8 @@ There are also several built-in variables:
  * The maximum values for the different integer types are available as the
    variables `max_uint`, `max_address`, `max_uint8`, `max_uint16` etc.
 
-CVL also has three built-in functions for casting mathematical types:
-`to_uint256`, `to_int256`, and `to_mathint`.  See {doc}`mathops` for details.
+CVL also has several built-in functions for converting between
+numeric types.  See {doc}`mathops` for details.
```

Comment:
Maybe we should put a big note at the top of `mathops` temporarily redirecting to the CVL 2 change guide?

Reply:
Uh, I didn't *really* mean to get dragged into updating the rest of the expression documentation, this was all best effort.